module/src/plugin/components/UserPicker.tsx

```diff
@@ -12,8 +12,8 @@ export const UserPicker = (props: UserPickerProps) => {
   return (
     <TypeAheadSelectField
       {...props}
-      label="User"
-      items={users.map(({ displayName, itemId }) => ({ value: itemId, displayText: displayName }))}
+      label={props.label}
+      items={users.map(({ displayName }) => ({ value: displayName, displayText: displayName }))}
```

Comment:
Hi Ketan - why do we use the displayName as the value here?   

The displayName is volatile, and can change on the backend.   
If this happens, then we lose the association to the user.   
The user's itemId does not change, so it's a stable association property.

Reply:
hay @ketankanhasoft 
https://www.awesomescreenshot.com/video/6956858?key=1a0445b26df230a3f0ae59938cf41faa

when I click on user drown down and select value as admin, so in this need to show name admin in the dropdown but showing it's itemId so in this case, I have change value from itemId to displayName.

Reply:
We may need another way to solve this 🤔 
Can you research a bit more?   

We need the displayname to show in the select box, but we need to pass the user's id [itemId] as the value to associate with the group.  🙏